en/release-notes/upgrade-guide-v3.0.md

```diff
@@ -314,14 +431,138 @@ Notice from the above two figures, there is a big difference between v2.4.3 and
 
 5. Some resources needed for compilation, such as application icons, application startup scripts, etc., v2.4.3 are stored in the build project, while v3.0 are stored in the `native/engine/name of the currently built platform` directory (e.g.: `native/engine/win32`, `native/engine/android`).
 
-## TypeScript Reference Tutorial
+## Upgrade FAQ
+
+### After the upgrade, the project script shows errors when opening in VS Code with operations such as binding component definitions
+
+Cocos Creator 3.x enables Strict Mode for TypeScript, which will review the code more strictly and eliminate any problems that may occur due to negligence during the development process.
+
+If you don't want to use strict mode, you can check **Enable loose mode** in **Project -> Project Settings -> Scripting** in the top menu bar of Creator. As a reminder, we do not encourage turning off strict mode, as strict null checking can reduce some low-level errors in the code runtime.
```

Comment:
```suggestion
To avoid using strict mode, check **Enable loose mode** in **Project -> Project Settings -> Scripting** in the top menu bar of Creator. As a reminder, turning off strict mode is not encouraged as strict null checking can reduce some low-level errors in the code runtime.
```